src/applications/bmqtool/m_bmqtool_interactive.cpp

```diff
@@ -147,6 +151,14 @@ void Interactive::printHelp()
         << bsl::endl
         << "    - 'post' command requires 'uri' and 'payload' arguments, "
         << "parameter 'messageProperties' is optional" << bsl::endl
+        << bsl::endl
+        << "  batch-post uri=\"bmq://bmq.test.persistent.priority/qqq\" "
+           "payload=[\"sample message\"] eventsCount=300 postRate=10 "
```

Comment:
Might be also good to introduce optional params in the same order as in `--help` docs